token-stakedrop/README.adoc

```diff
@@ -1,70 +1,87 @@
-# KEEP Token Tracker
+# Overview
 
-This is a tool for tracking KEEP Token ownership. It will walk over predefined
-sources of truth and inspect KEEP Token ownership at a specific block number.
+This is a tool for finding KEEP token owners on Ethereum (or on any other 
+EVM-compatible chain). It will walk over predefined sources of truth
```

Comment:
It iterates over sources of truth (predefined contracts and individual accounts) to find KEEP token owners at a target block number.

Reply:
In this case, source of truth is one of the components looking up the information. We have a separate source of truth for KEEP liquid balances, separate source for token grants and so on. This is how @nkuba's code structured it.

Reply:
https://github.com/keep-network/keep-core/tree/master/token-stakedrop/src/truth-sources

Reply:
Yea, I know that, that's why I'm saying "it iterates over sources of truth.. "

Reply:
Ah, I missed the suggestion in the diff. Will change.

Reply:
Ended up with https://github.com/keep-network/keep-core/pull/2400/files#diff-a0b8a21602c359b9faff24ed6a5b2514bd7e4360b7c483756e817034b961107a